Druid-Tests/TestAsserter.extension.st

```diff
@@ -0,0 +1,7 @@
+Extension { #name : 'TestAsserter' }
+
+{ #category : '*Druid-Tests' }
+TestAsserter >> onPharoCITestingEnvironment [
+
+	^ Smalltalk os environment at: 'PHARO_CI_TESTING_ENVIRONMENT' ifPresent: [ true ] ifAbsent: [ false ]
+]
```

Comment:
I moved this from the subclass, but I must add it as an extension until making a PR to Pharo.